profilegen/src/res/configurable.rb

```diff
@@ -0,0 +1,73 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+module Azure::ARM
+  # The Azure::ARM::Configurable module provides basic configuration for Azure ARM activities.
```

Comment:
If this is being put under the namespace `Azure::ARM::` Would the top level gem `azure_sdk` have namespace conflict? https://github.com/Azure/azure-sdk-for-ruby/blob/master/azure_sdk/lib/azure_sdk/configurable.rb#L5

Reply:
Changed the namespaces to Azure::Profiles::Management. Is it Management/ARM? That is open to discussion

Reply:
If we're looking to close on discussion on some points for this review, please set something up.